### PR TITLE
Giskard custom feedback ui

### LINKED ISSUE #2704: Custom feedback UI
iOS version here: #2626

### PR DESCRIPTION
**Pull request by daividssilverio**
## What's this?
The custom feedback ui implementation for the android app.
It's still missing: 
- [x] a little design tweak about the error view;
- [x] the success toast

### Relationships
Android version of #2626
Closes #2704

## Why do we want this?
Because it looks better than a simple e-mail.

## How is it done?
Well, it's quite straightforward. I'm simply binding the existing ViewModel observables to widgets.

### Why not another way?
It looks pretty simple this way. I'm open to suggestions.

### Side effects
The old feedback interactor becomes obsolete. We won't be sending e-mails for feedback purposes anymore.

## Review hints
https://github.com/toggl/mobileapp/pull/2710#issuecomment-408353346 can help testing errors.

## :squid: Permissions
~Not yet. WIP at the moment. I forgot to add the toast at the end.~
~Anyone after that.~
🦑 o/

### CODE REVIEW

File: Toggl.Foundation.MvvmCross/ViewModels/Settings/SendFeedbackViewModel.cs [hunk 1]
```diff
@@ -22,6 +22,7 @@ public sealed class SendFeedbackViewModel : MvxViewModelResult<bool>
         // Internal States
         private readonly ISubject<bool> errorViewVisibleSubject = new BehaviorSubject<bool>(false);
         private readonly ISubject<bool> isLoadingSubject = new BehaviorSubject<bool>(false);
+        private readonly ISubject<Exception> currentErrorSubject = new Subject<Exception>();
```
Comment (khoi) [inline Toggl.Foundation.MvvmCross/ViewModels/Settings/SendFeedbackViewModel.cs]: this is duplication of `errorViewVisibleSubject`, we can remove `errorViewVisibleSubject` if you want to know what the exception is

**Comment by daividssilverio** [comment]
I've added the toast and fixed the tests.

**Comment by khoi** [comment]
Checked Daneel, works ok with the new property
Giskard is broken 💥

**Comment by daividssilverio** [comment]
I moved a string to the strings.xml file and forgot to escape the `'`... 🤦‍♂️

**Pull request merged**